README.md

```diff
@@ -5,3 +5,44 @@ Adds automated code upgrades to REDAXO improving developer productivity and code
 The AddOn integrates [rector](https://github.com/rectorphp/rector) with the developer in mind, meaning it eases use for often used migration/upgrade use-cases. If you are a rector expert you don't need this AddOn. 
 
 Primary purpose is to allow people less experienced with developer tooling to automate migration tasks.
+
+Users apply the changes after controlling the changes in a diff view.
+
+## use cases
+
+### PHP Version Migrations
+
+This use case helps updating the PHP version used in a project to a newer one. This can involve migrating code to be compatible with the new version and updating any deprecated features to the recommended replacements.
```

Comment:
```suggestion
This use case helps updating the PHP version used in a project to a newer one according to the office php.net migration guide. This can involve migrating code to be compatible with the new version and updating any deprecated features to the recommended replacements.
```